packages/core-data/src/queried-data/selectors.js

```diff
@@ -124,9 +124,3 @@ export function getQueriedTotalItems( state, query = {} ) {
 
 	return state.queries?.[ context ]?.[ stableKey ]?.meta?.totalItems ?? null;
 }
-
-export function getQueriedTotalPages( state, query = {} ) {
```

Comment:
There's another selector like that in the core-data/src/selectors, I guess we should remove it too.